### ISSUE TITLE
Find a New Home for the "Portability" Page from v2.ocaml.org

### ISSUE THREAD

**Issue by sabine** [ocaml/ocaml.org#1793]
On https://v2.ocaml.org/learn/portability.html, we have a page "Portability".

This information needs a new home on the current ocaml.org site.

**Comment by cuihtlauac** [comment]
I support adding it to the [Platform Tools](https://ocaml.org/docs/platform) section. It is a focus on a tool, the compiler.

**Comment by tmattio** [comment]
Indeed, that's an embarrassing oversight on my part, it's very important information to have on the site!

I agree with @cuihtlauac, the Platform section looks like a good place for this. There will be differences between what the compiler supports, the Platform tools and the opam infrastructure, so we might want to add the information for the platform tools and the infra as well.

**Comment by sabine** [comment]
Ok, so then this can go in Platform Tools > Compiler > Portability.
